src/configure

```diff
@@ -1410,6 +1412,7 @@ Optional Features:
   --enable-experimental-code
                           Use experimental code
   --enable-fuzz           Fuzzing prepare(), valid(), init(), etc
+  --enable-fuzzing        Fuzzing using libFuzzer
```

Comment:
Since the functionality is specific to libFuzzer, the option should be called --enable-libfuzz. We already also have --enable-fuzz for the builtin fuzzing (maybe rename it to --enable-builtin-fuzz?) and we'll probably have more of these later.